tests/settings.py

```diff
@@ -12,7 +12,8 @@
 SECRET_KEY = "!2*nm%ps%x8!ykyb^s9+!l1vcmeh+(f&de%br=js*7(5i_rmet"
 
 # needed since travis uses Ubuntu 14.04
-if distro.linux_distribution() == ('Ubuntu', '16.04', 'Xenial Xerus'):
+if distro.linux_distribution() == ('Ubuntu', '16.04', 'Xenial Xerus') or \
+    distro.linux_distribution() == (u'Linux Mint', u'18.3', u'Sylvia'):
```

Comment:
Interesting

Reply:
yeah it wouldn't use the sqlite geo extension for me without this...

Reply:
So, is not a general rule, is a specific setting tailored on your system's characteristics, what about having a simple flag instead?

Reply:
Hard to tell, seem at least these older Ubuntus need the variable `SPATIALITE_LIBRARY_PATH` set. Hard to tell if it breaks something for another distros.

https://code.djangoproject.com/ticket/26137

Reply:
ok, thanks